modules/command/src/main/java/org/apache/fluo/command/ConfigOpts.java

```diff
@@ -16,14 +16,23 @@
 package org.apache.fluo.command;
 
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 import com.beust.jcommander.Parameter;
+import com.beust.jcommander.converters.IParameterSplitter;
 import org.apache.fluo.api.config.FluoConfiguration;
 
 public class ConfigOpts extends BaseOpts {
 
-  @Parameter(names = "-D",
+  public static class NullSplitter implements IParameterSplitter {
+    @Override
+    public List<String> split(String value) {
+      return Collections.singletonList(value);
+    }
+  }
+
+  @Parameter(names = "-D", splitter = NullSplitter.class,
```

Comment:
I think `-o`, or `-O`, or `--option` would be less confusing here, because `-D` conflicts with java system parameters. I realize that's not part of this PR, but it's something I noticed while examining this PR which could be changed in a follow-on issue.

Reply:
Made option `-o`